views/default/page/elements/by_line.php

```diff
@@ -0,0 +1,65 @@
+<?php
```

Comment:
why this file? we already have object/elements/imprint/byline

Reply:
Hm, merge conflicts